File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisAccessorDefinedException.java
package org.irislang.jiris.core.exceptions.fatal;

/**
 * Created by Huisama on 2017/5/26 0026.
 */
public class IrisAccessorDefinedException extends IrisFatalException {
    public IrisAccessorDefinedException(String fileName, int lineNumber, String message) {
        super(fileName, lineNumber, message);
    }

    @Override
    public String GetFatalExceptionName() {
        return "AccessorDefinedIrregular";
    }
}

File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisAccessorDefinedException.kt
package org.irislang.jiris.core.exceptions.fatal

/**
 * Created by Huisama on 2017/5/26 0026.
 */
class IrisAccessorDefinedException(fileName: String, lineNumber: Int, message: String) : IrisFatalException(fileName, lineNumber, message) {

    override fun GetFatalExceptionName(): String {
        return "AccessorDefinedIrregular"
    }
}
